ddtrace/tracer/sampler_test.go

```diff
@@ -469,26 +470,28 @@ func TestSamplingLimiter(t *testing.T) {
 		assert.True(sampled)
 		assert.Equal(0.42, rate)
 		assert.Equal(now, sl.prevTime)
-		assert.Equal(100, sl.seen)
-		assert.Equal(42, sl.allowed)
+		assert.Equal(float64(100), sl.seen)
+		assert.Equal(float64(42), sl.allowed)
```

Comment:
Can we use `100.0` and `42.0` here rather than using conversion, for consistency?